02-peer-protocol.md

```diff
@@ -361,28 +355,27 @@ For the `minimum fee` calculation see [BOLT #3](03-transactions.md#calculating-f
 The maximum inputs and outputs are capped at 252. This effectively fixes
 the byte size of the input and output counts on the transaction to one (1).
 
-
 ### The `tx_signatures` Message
 
 1. type: 71 (`tx_signatures`)
 2. data:
     * [`channel_id`:`channel_id`]
     * [`sha256`:`txid`]
-    * [`u16`:`num_witnesses`]
+    * [`bigsize`:`num_witnesses`]
```

Comment:
Can you have more than 65k inputs? :raised_eyebrow:

Reply:
No, especially since we're limiting them to 252 in the requirements section, but it's more consistent to start using `bigsize` everywhere, isn't it? Also here it means we'll always use a single byte (because we don't allow more than 252 inputs) instead of a two bytes (granted, we don't really care about 1-byte bandwidth optimizations, but :nerd_face:).